commonutils/src/test/java/io/intrepid/commonutils/IntrepidViewUtilsTest.java

```diff
@@ -25,6 +37,28 @@
 public class IntrepidViewUtilsTest {
 
     @Test
+    public void testSetVisibilities() throws Exception {
+        final View view1 = new View(RuntimeEnvironment.application);
+        final View view2 = new View(RuntimeEnvironment.application);
+        final View view3 = new View(RuntimeEnvironment.application);
+
+        setVisibilities(View.VISIBLE, view1, view2, view3);
```

Comment:
could have utility method, which asserts that given views have given/expected visibility